tests/integration_tests/analytics/test_get.py

```diff
@@ -94,28 +94,6 @@ def test_get_analytics_summary_unfccc(
     }
 
 
-def test_get_analytics_summary_other(
```

Comment:
Is this just a drive by removal, I can't see the test replaced anywhere?

Reply:
Yeah it's no longer necessary.